webapp/src/app/query-runner/query-runner.component.html

```diff
@@ -222,7 +222,31 @@
         </p>
 
         <p *ngIf="predict === undefined">Loading dataset's columns...</p>
-        <form
+        <table mat-table *ngIf="predict !== undefined">
+          <tr>
+            <ng-container *ngFor="let field of predict.fields">
+              <th mat-header-cell class="mat-header-cell cdk-header-cell">
+                {{ field.name }}
+              </th>
+            </ng-container>
+          </tr>
+          <tr>
+            <ng-container *ngFor="let field of predict.fields">
+              <td mat-cell class="mat-cell cdk-cell">{{ field.value }}</td>
+            </ng-container>
+          </tr>
+          <!--<th mat-header-cell *matHeaderCellDef> {{ field.name }} </th>
+            <td mat-cell *matCellDef>{{ field.value }}</td>-->
+
+          <!--<tr mat-header-row *matHeaderRowDef="field. "></tr> 
+          <tr mat-row *matRowDef="let field; columns: columnsToDisplay"></tr>s-->
+        </table>
+
+        <button mat-raised-button color="primary" (click)="runPredictRequest()">
+          Predict
+        </button>
+
+        <!--<form
```

Comment:
Also remove?

Reply:
Yep, sorry about this, I'll do my best to push more "polished" PR